controls/cis_rhel7.yml

```diff
@@ -965,3 +965,245 @@ controls:
+    levels:
+    - l1_server
+    - l1_workstation
+    automated: no # rule missing
+
+  - id: 3.5.1.4
+    title: Ensure firewalld service enabled and running (Automated)
+    levels:
+    - l1_server
+    - l1_workstation
+    automated: partially # only checking systemd service, not running "firewall-cmd --state"
+    rules:
+    - service_firewalld_enabled
+
+  - id: 3.5.1.5
+    title: Ensure firewalld default zone is set (Automated)
+    levels:
+    - l1_server
+    - l1_workstation
+    automated: no # no exact rule is present
+
+  - id: 3.5.1.6
+    title: Ensure network interfaces are assigned to appropriate zone (Manual)
+    levels:
+    - l1_server
+    - l1_workstation
+    automated: no
+
+  - id: 3.5.1.7
+    title: Ensure firewalld drops unnecessary services and ports (Manual)
+    levels:
+    - l1_server
+    - l1_workstation
+    automated: no
+
+  - id: 3.5.2.1
+    title: Ensure nftables is installed (Automated)
+    levels:
+    - l1_server
+    - l1_workstation
+    automated: no # rule missing
+
+  - id: 3.5.2.2
+    title: Ensure firewalld is either not installed or masked with nftables (Automated)
+    levels:
+    - l1_server
+    - l1_workstation
+    automated: no # rule missing
+
+  - id: 3.5.2.3
+    title: Ensure iptables-services not installed with nftables (Automated)
+    levels:
+    - l1_server
+    - l1_workstation
+    automated: no # rule missing
+
+  - id: 3.5.2.4
+    title:  Ensure iptables are flushed with nftables (Manual)
+    levels:
+    - l1_server
+    - l1_workstation
+    automated: no
+
+  - id: 3.5.2.5
+    title: Ensure an nftables table exists (Automated)
+    levels:
+    - l1_server
+    - l1_workstation
+    automated: no # rule missing
+
+  - id: 3.5.2.6
+    title: Ensure nftables base chains exist (Automated)
+    levels:
+    - l1_server
+    - l1_workstation
+    automated: no # rule missing
+
+  - id: 3.5.2.7
+    title: Ensure nftables loopback traffic is configured (Automated)
+    levels:
+    - l1_server
+    - l1_workstation
+    automated: no # rule missing
+
+  - id: 3.5.2.8
+    title: Ensure nftables outbound and established connections are configured (Manual)
+    levels:
+    - l1_server
+    - l1_workstation
+    automated: no
+
+  - id: 3.5.2.9
+    title: Ensure nftables default deny firewall policy (Automated)
+    levels:
+    - l1_server
+    - l1_workstation
+    automated: no # rule missing
+
+  - id: 3.5.2.10
+    title: Ensure nftables service is enabled (Automated)
+    levels:
+    - l1_server
+    - l1_workstation
+    automated: no # rule missing
+
+  - id: 3.5.2.11
+    title: Ensure nftables rules are permanent (Automated)
+    levels:
+    - l1_server
+    - l1_workstation
+    automated: no # rule missing
+
+  - id: 3.5.3.1.1
+    title: Ensure iptables packages are installed (Automated)
+    levels:
+    - l1_server
+    - l1_workstation
+    automated: yes
+    # rules:
+    # - package_iptables_installed
```

Comment:
Any reason why this rule is commented?
Is this commented because it is repeated? The control format is able to handle repeated rules just fine.

Reply:
This item is missing `iptables-services' installed too.